custom_components/tuya_local/devices/madimack_elite_v3_heatpump_updated.yaml

```diff
@@ -0,0 +1,200 @@
+name: Madimack Elite V3 pool heatpump

```

Comment:
Ideally this config should have a `products` section listing the `id` and branding part of the `name`, and this top level name should just be the generic part: "Pool heatpump".

Reply:
Yes, I can find the product ID through the tuya platform and change. However, I have noticed the product ID code found is identical to another device by the same manufacturer already in your system (madimack_eco_heatpump.yaml). Will using the same product ID code for a different unit be a problem?

Reply:
I imagine when I eventually use this, it will be capable of showing a list, the same as the dps matching that is currently used (and will continue to be used as a backup).  But if the product ids are the same, maybe the config is also compatible (maybe only partially specified, as the knowledge of where to get good info from to make a complete config has evolved over time).

Reply:
I have added a product ID line and the specific name of the unit I am using to my fork. I'm also a github newbie so let me know if you cannot see this.

As mentioned before - there may be an issue in that the product ID appears to be duplicated with other completely separate products made by the same manufacturer